### ISSUE TITLE
Translator window doesn't pop up in cutscenes

### ISSUE THREAD

**Issue by Murkchan** [lokinmodar/Echoglossian#5]
Translator window (in no replace mod) doesn't pop up in cutscenes.

**Comment by lokinmodar** [comment]
About the 1st topic: It is in my TODO list. It is being reworked in the new Dalamud API branch and hopefully it will be fixed.

About the 2nd topic: There are some in-game texts like that from the screenshot that are untranslatable because they are images instead of real text lol. (Thanks S.E.)  Unless we come up with a way to index those lines and translate those texts independently and attach a translation window to them, unfortunately they will remain like that. If someone can help with this topic it will be greatly appreciated <3

**Comment by Murkchan** [comment]
> Unless we come up with a way to index those lines and translate those texts independently and attach a translation window to them, unfortunately they will remain like that. If someone can help with this topic it will be greatly appreciated <3

Tataru Helper already found how to index.

**Comment by lokinmodar** [comment]
> > Unless we come up with a way to index those lines and translate those texts independently and attach a translation window to them, unfortunately they will remain like that. If someone can help with this topic it will be greatly appreciated <3
> 
> Tataru Helper already found how to index. Probably these replicas are duplicated in the event chat.

Yes. They rely on the chat event which I chose not to for this plugin. 

Still there are some cutscene texts that do not get posted to chat. Also, I'm not copying their code unless they give me permission to. My aim with this project is not having to rely on an independent program to do the job. Their program requires you to run the game in Borderless Windowed (which can impact performance) and Dalamud does not. But if I (or a helping hand) come up with a solution that complies with Dalamud i'll implement it of course.

**Comment by lokinmodar** [comment]
Implemented in v3.7.x
